packages/sdk/src/sdk/api/generated/full/models/CoverPhotoFull.ts

```diff
@@ -0,0 +1,82 @@
+/* tslint:disable */
+/* eslint-disable */
+// @ts-nocheck
+/**
+ * API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+/**
+ * 
+ * @export
+ * @interface CoverPhotoFull
+ */
+export interface CoverPhotoFull {
+    /**
+     * 
+     * @type {string}
+     * @memberof CoverPhotoFull
+     */
+    _640x?: string;
```

Comment:
Late to the game. Are they always optional or should we just make the entire object optional and the fields inside required?
The parsing of these on client is kind of annoying when you have to check the outer object and then check everything inside it. But if there truly is a use case where some of them would be defined and others would not, I'm cool with it.

Reply:
good comment. we should be able to update this. will track in a ticket. can fix non-full too.